src/main/webapp/editFilesFragment.xhtml

```diff
@@ -70,7 +94,7 @@
                                   disabled="#{!(datasetPage || EditDatafilesPage.showFileUploadComponent())}"
                                   fileUploadListener="#{EditDatafilesPage.handleFileUpload}" 
                                   process="filesTable" 
-                                  update="@this" 
+                                  update=""
```

Comment:
The comment above is extremely useful and helps out with #4549. I was curious about when `""` got changed to `"@this" ` and it looks like it was 57db2f9 which was part of the fairly gigantic "file replace" pull request in #3579.